### DISCUSSION TITLE
Icebergs

### DISCUSSION THREAD

**Discussion by mankoff** [NASA-GISS/freshwater-forcing-workshop#19] [Timeseries development]
From Ben Davison:

> [...] what is the desired horizontal distribution and format of the freshwater anomaly dataset? [...] Point-source or basin-integrated data are more readily available for all freshwater sources than equivalent gridded data describing those fluxes at the ice-ocean boundary.
>
> For freshwater sourced from submarine melting, is the expectation/hope that that the freshwater dataset will somehow incorporate the effect of latent heat, or will that be dealt with at runtime?
>
> Similar to unresolved cavity circulation mentioned by Tore, fjords are also unresolved. Donald Slater presented an approach and dataset to account for that, but this is a very different dataset to the point source runoff from e.g. Mankoff et al. (2020) and it's not clear to me which is preferred here.

From @gavinny

> The horizontal spatial resolution of the map over which freshwater from iceberg fluxes would be distributed should be commensurate with ocean model resolutions – so 1º at minimum, but 0.25º might be good. Anything higher is probably overkill?

> With respect to the spatial resolution of the icesheet fluxes, we can probably group that by major basin watershed (iceshed?) – the details of which the ice sheet folk can determine, but I would expect half a dozen or so independent time-series for each ice sheet (e.g. 6 for Greenland, 6 for Antarctica – or whatever the natural agglomeration looks like). Greater granularity is probably wasted on relatively coarse resolution ocean models, but eventually might be useful.

> For basal ice shelf melt, and for iceberg melt, we will definitely recommend taking the latent heat for melting from the ocean (so the freshwater will be accompanied with a negative heat flux. We could also include a (smaller) sensible heat flux as well if people felt that if the ice in the icebergs is significantly colder than 0ºC.

> With respect to the cavities and fjords, we can mostly assume that the ocean models start at the iceberg front or fjord mouth, and so the net effects are what are needed (which will have a different distribution in the vertical). Some models will start resolving specific cavities (and maybe fjords) relatively soon, and so we can advise on some ‘mix and match’ approaches – for instance, if basal melt in a cavtity is calculated, one would not want the imposed FW flux from basal melt from that basin.

From Frank Pattyn:

> As an ice sheet modeller I can briefly comment that for the Antarctic ice sheet the easiest way to proceed is to use ice sheet basins (27 or so exist in either the Zwally and Rignot basins), which is also easy to group them if you wish to have information of FWFs on a coarser level. This basins correspond to outlets of approximately 10°. Other output could be grid cell output, but that may be more complicated as calving fluxes are also vectors and you need to conserve mass in the regridding process. Also grid size is different for each model and may vary between 1 and 10km or more. It is also easy to provide mean ice shelf thickness, which may used as an indicator of at what depth meltwater fluxes enter the ocean. Again, subshelf melt can also be provided on a grid basis. Of course I am talking about model results and not observations, but these are either across the historical period or for projections. Cheers,

**Comment by mankoff** [comment]
## Heat


<a id="org792c00f"></a>

### Latent heat

Latent heat: I suggest deal with it at runtime. A data set could include the latent heat necessary to melt that volume of freshwater, but calculating that is trivial, and I think the ocean model will need to do the heavy lifting to properly incorporate that value.


<a id="org9b6fb23"></a>

### Sensible heat

There are two components to this. Warming ice to the phase transition temperature (PTT), and then warming the freshwater from the PTT (may be up to -2) to the ambient ocean temperature. This may be up to 15 % of the heat budget so I think they should be handled. Ice warming needs to be assumed and parameterized by an ocean model. Warming the meltwater to ambient may be explicit or implicit in how the freshwater forcing is handled depending on the model?

-   Ice shelves are generally -20 °C (<http://doi.org/10.3189/2013JoG12J207> Fig. 2).
-   Ocean thermal forcing can be up to 4 °C (<http://doi.org/10.1038/ncomms14507>).

Order of magnitude,

```python
from pint import UnitRegistry
u = UnitRegistry()

c_i = 2093 * (u.J / (u.kg * u.K))	# specific heat capacity of ice
L = 334E3 * (u.J / u.kg)		# latent heat of fusion for water
k = 4180  * (u.J / (u.kg * u.K))	# specific heat capacity of water

m = 1 * u.kg				# sample mass
T_0 = (273.15 - 2) * u.K		# phase transition temperature
T_i = (273.15 - 20) * u.K		# ice temperature @tyler_2013
T_w = T_0 + (4 * u.K)			# ocean temperature @webber_2017

Q_warm_ice = (m * c_i * (T_0 - T_i))
Q_melt_ice = (m * L)
Q_warm_water = (m * k * (T_w - T_0))

Q_tot = Q_warm_ice + Q_warm_water + Q_melt_ice

print(f"Warm ice from {T_i.to(u.degC)} to {T_0.to(u.degC)}: {Q_warm_ice} ({(Q_warm_ice/Q_tot).to('%').round(1)})")
print(f"Melt ice: {Q_melt_ice} ({(Q_melt_ice/Q_tot).to('%').round(1)})")
print(f"Warm water from {T_0.to(u.degC)} to {T_w.to(u.degC)}: {Q_warm_water} ({(Q_warm_water/Q_tot).to('%').round(1)})")

```

```python
: Warm ice from -20.0 degree_Celsius to -2.0 degree_Celsius: 37674.0 joule (9.7 percent)
: Melt ice: 334000.0 joule (86.0 percent)
: Warm water from -2.0 degree_Celsius to 2.0 degree_Celsius: 16720.0 joule (4.3 percent)
```

<a id="orgabf3ca0"></a>

## File format

The modelers should drive the **final** format and details (CSV, NetCDF (grid or vector), approximate resolution if gridded, map projection, etc.), assuming there is even consensus between models.


<a id="orgc8308e9"></a>

### Subglacial discharge (surface and subglacial melt) and sub-shelf discharge

For liquid discharge (sub-shelf melt and surface runoff) I think both Frank and Gavin are suggesting vector format, not gridded. I agree. See comments below on binning by sector/region.


<a id="org02958f8"></a>

### Icebergs

I think solid discharge should be provide as point-source (per glacier, or perhaps per region) because some model oceans may want to route and melt their own icebergs.

We can also provide a spatial map of where icebergs are expected to melt, but I'd like input from others on how to produce this map. Maps of iceberg locations exist (see <https://www.scp.byu.edu/data/iceberg/A68tracking.html> ) but this is a form of [survivorship bias](https://en.wikipedia.org/wiki/Survivorship_bias). They melt elsewhere. Elsewhere is not as simple as "icebergs flow north", so icebergs in a given sector may be sourced from other longitudes.

I'm not sure what to do about this, and suggest a separate topic to discuss freshwater from icebergs (sources, end-points, melt rate, melt depth).

Icebergs (like melting shelf ice, see above) should probably be at -10 or -20 C, unless Peninsula glaciers have warmer temperature profiles.


<a id="orgede0a3a"></a>

### Spatial resolution (of vector data)

I suggest the data providers should generate the highest fidelity product they can (assuming higher temporal and spatial resolution does not impose undue effort burden on the work).

The benefit of producing the data with higher fidelity is more people will use it outside of this project/community. I have a lot of observationalists and process study people using products that provide glacier- or fjord- scale spatial resolution, but it cost me no effort relative to e.g. 'Zwally sector' which might be all that a GCM wants or needs.

The method to bridge between high-fidelity data and model input is a) choosing the right format b) extensive use of METADATA, and c) example code (or 'downstream products').

As one example, my 'Greenland freshwater discharge' product (<http://doi.org/10.5194/essd-12-2811-2020>) provides daily temporal resolution and stream-scale vector (not gridded) spatial resolution. There are 29430 'stations' (stream outlets). I assume this is worthless as model input. However, for every station I provide the following metadata:

-   Zwally 2012 sector
-   Mouginot 2019 ID (and distance&#x2026;)
-   Mouginot 2019 basin
-   Mouginot 2019 region
-   Mankoff 2020 gate ID
-   Mankoff 2020 gate distance
-   Bjørk 2015 name
-   And other data (elevation, lat, lon, distance from station to nearest edge of each of the regions above, etc.)

These 29430 stations at daily resolution can then be collapsed to annual runoff per Zwally basin (as exampled by Frank) with just a few lines of code. Data producers can do that as part of this effort, or not, if final use cases are TBD. When someone asked me how make a downstream product, in addition to doing it for them, I document it in the README or on the GitHub discussion page for that product (See README at  <https://github.com/GEUS-Glaciology-and-Climate/freshwater/> and reduce-by-ROI example at <https://github.com/GEUS-Glaciology-and-Climate/freshwater/discussions/34>)

**Reply by trackow** [reply]
As was mentioned above by @mankoff, the spatial distribution of iceberg melt around Antarctica goes beyond a simple "icebergs flow north". In the absence of an interactive iceberg model, if we want to treat different point sources or IMBIE regions separately, we can unfortunately **not** just cut the hemispheric melt patterns from available iceberg melt climatologies (Merino et al, Jourdain et al., Rackow et al, and others) into different disjunct sectors, normalise them, and multiply by the wanted amount from the nearby region: To give an example, the melt pattern for the Weddell Sea is partly the result of icebergs stemming from the opposite site, the Ross Sea. 

To tackle this, C. Rodehacke (Danish Meteorological Institute) and I had discussed the following approach some years ago (see https://github.com/trackow/iceberg-templates/blob/master/README.md):

- define major iceberg release regions based on the IMBIE regions (see our discussion here https://github.com/trackow/iceberg-templates/blob/master/README.md for why we think 10 regions could be a good choice)

- based on trajectory output from numerical iceberg-ocean-sea ice models (this could be replaced potentially by observed iceberg location data from BYU or other sources), identify trajectories that start within the release regions and create spatio-temporal iceberg patterns or PDFs from it (such that those integrate to 1). This allows us to multiply by the wanted integrated amount of iceberg melt originating from these regions, and the PDFs will distribute this amount correctly in space

- The result is 10 melt patterns for the individual release regions that can be included in other models (that don't include an interactive iceberg drift and decay model) in order to distribute the iceberg melt from a particular release region into other neighbouring and remote regions, occasionally at opposite sites of Antarctica (because giant bergs can travel in the coastal current and survive there for years).


The disadvantage is that the provided regional templates are model-based at the moment, but I think most of the listed hemispheric climatologies above are as well.

**Reply by nicojourdain** [reply]
Hi Thomas,

I agree this is the right approach. Nacho Merino actually coded that in NEMO during his PhD (for 10 sectors as well but not defined based on IMBIE basins), and I used it in my 2019 update of his dataset. Here is the kind of melt pattern that we have (the rectangle shows the origin of icebergs):

Here for the mean melt rate of icebergs from the Eastern Amundsen Sea :
<img width="681" alt="Screenshot 2024-04-30 at 16 07 23" src="https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/25052687/976a536d-3d36-49b0-a77a-274266ac7fb4">

Here for the mean melt rate of icebergs from the Larsen & Filchner Ronne area :
<img width="681" alt="Screenshot 2024-04-30 at 16 10 18" src="https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/25052687/190200d8-e6e1-417a-a580-3ec11a16bee9">

I think it would be better to redo this with the actual IMBIE basins and with a more realistic coastline (this one is cut at 77°S), but this one is available now if needed.

**Reply by mankoff** [reply]
Hi @trackow - this morning I did this for observations using the BYU database. This could be used for this project, or for validation.

See code and initial results at https://github.com/mankoff/iceberg_melt currently gridded at 100 km resolution.

The BYU database has a column for iceberg size, meaning we can get melt quantity between observations. But that field is unfortunately mostly blank, so the maps I've generated are iceberg position density, not melt rate.

**Reply by crodehacke** [reply]
I like this idea. I'm wondering if we could generalize it further and create related release patterns for longitude bins. In these cases, we could determine the related release patterns for different drainage basins without defining these basins upfront (in detail). This would allow us to make it consistent with basins used in the PICO model (https://dx.doi.org/10.5194/tc-12-1969-2018) to determine basal melting rates in PISM, for instance.

**Reply by mankoff** [reply]
Hi @crodehacke 

GitHub only allows 1 level of reply-depth so I'm not sure what you refer to when you say "I like this idea" - there are a few ideas discussed above.

By 'release pattern' do you mean meltwater release? Or iceberg release? Maybe it doesn't matter. Ideally meltwater, but if all we have is obs "berg position", then work with that.

My interpretation of 'release pattern by longitude bin (w/o defining basins upfront)' is to do what we've done above (obs or model), but instead of having 18 (or 10) IMBIE basins, have, say, 360 "basins", one per degree of longitude. I like this idea too. As I wrote [above](https://github.com/NASA-GISS/freshwater-forcing-workshop/discussions/19#discussioncomment-8973512), include metadata for each of these bins (IMBIE basin, Mouginot basin, shelf name, etc.) to make it easy for yet-to-be-determined downstream use-cases to be able to easily work with the data. It would be easy to resample to 18 basins, or 10, or whatever. This is one line of code thanks to xarray, if the metadata is there.

What resolution? Could be native model resolution, although a 1 minute ocean model would then generate 21600 maps. Most cells would be 0 and it would compress well...

**Reply by gavinny** [reply]
We discussed the resolution of the maps, and given the necessary smoothing and the imprecision of the modeling, I'm not persuaded that anything better than 1º or maybe 0.5º is really warranted.

**Reply by crodehacke** [reply]
I haven't dreamed of archiving such a high resolution of 1° or even finer. I consider several degrees a good choice for reasons listed by @gavinny, for instance.

**Reply by mankoff** [reply]
I've spent a day dividing Antarctic iceberg by 1 ° longitude bins, using the first observed position of icebergs in the BYU DB, and generating 360 maps (1 per initial location). This fails in some locations because Antarctica isn't a circle. The same initial longitude represents different IMBIE basins around the peninsula and near the Western edge of the Ross shelf and Victoria Land.

This type of organization is even more complex in Greenland.

For now, I think I'll return to dividing by Rignot (IMBIE) basins.

FYI, here's 10 random lines from a CSV containing all iceberg observation positions summed by days-in-1°x1°-bin:

```csv
east         north        cat     lon       lat      yyyydoy      id     size      basin_cat  basin_name  lon0
-1065414.37  990390.7     36631   -47.090   -76.670  2022295.000  a74a   0.000     14         K-A         -36.295
2459803.92   -1104602.58  195242  114.183   -65.546  2006121.000  c19c   1092.670  11         Dp-E        166.300
-2548641.18  1849861.84   105323  -54.027   -61.589  2015172.000  b15x   0.000     12         Ap-B        45.760
1320941.53   -2048208.06  176255  147.181   -67.838  2005264.000  c15    0.000     5          D-Dp        147.122
-2388598.6   3241948.18   197747  -36.382   -54.112  2012255.000  c19c1  2808.130  17         I-Ipp       -35.762
-593211.98   -1283710.67  153353  -155.198  -77.038  2021142.000  b38    0.000     19         Ep-F        -155.177
2606081.87   -862434.27   56806   108.311   -65.118  2012358.000  b09d   459.390   5          D-Dp        143.260
-1366655.01  -3647326.21  90539   -159.459  -55.218  2011310.000  b15j   0.000     11         Dp-E        168.983
2766787.98   -304967.47   221364  96.290    -64.780  2023223.000  c30    0.000     13         C-Cp        96.284
2179555.57   -1631346.66  274586  126.814   -65.317  2013255.000  uk192  0.000     6          Cp-D        131.788
```

And an initial NetCDF might look like this:

```
<xarray.Dataset>
Dimensions:       (lon0: 360, lat: 51, lon: 360)
Coordinates:
  ,* lon0          (lon0) int16 -180 -179 -178 -177 -176 ... 175 176 177 178 179
  ,* lat           (lat) int16 -90 -89 -88 -87 -86 -85 ... -44 -43 -42 -41 -40
  ,* lon           (lon) int16 -180 -179 -178 -177 -176 ... 175 176 177 178 179
    spatial_ref   int64 0
Data variables:
    iceberg_days  (lon0, lat, lon) float64 nan nan nan nan ... nan nan nan nan
    rignot_label  (lon0) <U6 '      ' 'Dp-E' '      ' ... '      ' '      '
    rignot_id     (lon0) int16 0 11 0 4 0 0 0 0 0 0 19 ... 11 4 11 0 0 0 0 0 0 0
    overview      (lat, lon) float64 0.0 0.0 0.0 0.0 0.0 ... 0.0 0.0 0.0 0.0 0.0
```

But as I said, I think I'll return to IMBIE regions, so instead of the first dimension being `lon0 = 360`, it might be `basin_id = 18`. Will post sample data soon for GL and AQ to see if this works for people.

**Comment by gavinny** [comment]
Yes. This is what I thought we’d agreed to do. However, I’m a little confused as to the number of IMBIE basins we are planning for. There are 17 on the IMBIE webpage, but maybe 10 is better – presumably this just amalgamates some of the smaller ones. We will need to smooth these fields a little for implementation within coarser resolution OGCMs but I don’t think that is problematic. Happy to work with whatever we have available.

Gavin

--
Gavin A. Schmidt
Director, NASA Goddard Institute for Space Studies
2880 Broadway, New York NY 10025
Cell: 212 749 0006
Email: ***@***.******@***.***>
URL: https://www.giss.nasa.gov/staff/gschmidt


From: Nicolas C. Jourdain ***@***.***>
Date: Tuesday, April 30, 2024 at 10:21 AM
To: NASA-GISS/freshwater-forcing-workshop ***@***.***>
Cc: Schmidt, Gavin A. (GISS-6110) ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] [BULK] Re: [NASA-GISS/freshwater-forcing-workshop] File formats (Discussion #19)
CAUTION: This email originated from outside of NASA.  Please take care when clicking links or opening attachments.  Use the "Report Message" button to report suspicious messages to the NASA SOC.



Hi Thomas,

I agree this is the right approach. Nacho Merino actually coded that in NEMO during his PhD (for 10 sectors as well but not defined based on IMBIE basins), and I used it in my 2019 update of his dataset. Here is the kind of melt pattern that we have (the rectangle shows the origin of icebergs):

Here for the mean melt rate of icebergs from the Eastern Amundsen Sea :
Screenshot.2024-04-30.at.16.07.23.png (view on web)<https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/25052687/976a536d-3d36-49b0-a77a-274266ac7fb4>

Here for the mean melt rate of icebergs from the Larsen & Filchner Ronne area :
Screenshot.2024-04-30.at.16.10.18.png (view on web)<https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/25052687/190200d8-e6e1-417a-a580-3ec11a16bee9>

I think it would be better to redo this with the actual IMBIE basins and with a more realistic coastline (this one is cut at 77°S), but this one is available now if needed.

**Reply by mankoff** [reply]
There are 18 basins. The IMBIE page says

> For the IMBIE 2024 assessment, we are aiming at producing reconciled time-series of mass changes for the 18 drainage basins of Antarctica

The graphic at http://imbie.org/imbie-3/drainage-basins/ has 18: 10 east, 5 west, 3 peninsula.

The data actually contains 19. There is one for "Islands".

**Reply by xylar** [reply]
In ISMIP6, we decided to combine basins that divide up ice shelves.  We ended up with 16 if I recall correctly.  It looks likely we would want to combine the Filchner and Ronne basins for purposes of generating icebergs and also the eastern and western Ross.  @nicojourdain, what are your thoughts here?

**Reply by nicojourdain** [reply]
Anna Olivé-Abello, working with Pierre Mathiot and myself at IGE, has actually already estimated the melt patterns from the 18 IMBIE basins. We can probably average over a longer period to have a smoother pattern. 
![MAP_NEW1v1_BedMachine_meltingbasin_lr](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/25052687/810833f0-0a03-4844-8895-d25cd05bffbd)

This is not perfect as this is a circum-Antarctic domain cut at 55°S, so we are missing ~70Gt/yr of melt in the South Atlantic, but this is probably the best we have in our group without having to wait for new simulations (and we can combine previous patterns to fix the South Atlantic leakage if needed).

**Comment by mankoff** [comment]
Should we be using observations (that provide iceberg density and time, but not melt), or model data?

Respond in comments or just vote by clicking on 👀   for Obs, or 👍🏿  for Model.

**Reply by gavinny** [reply]
model for completeness, obs for sanity checking.

**Reply by nicojourdain** [reply]
@xylar : in ISMIP6, we merged Ross and Filchner-Ronne because we wanted to calculate ice-shelf basal melting in a consistent way under these ice shelves, but for this work, there is probably no need to merge and we can keep 18 basins (I am not sure it will make a big difference anyway).

@mankoff : my question about the satellite data you have used is whether they represent only the largest icebergs or all the iceberg population? (I do not have time to go through the papers right now). What proportion of the total iceberg mass is represented. If only the largest icebergs are represented, this could give a slightly different melt pattern even though most of the mass is in the largest ones.

**Comment by mankoff** [comment]
@nicojourdain The BYU data comes from the US National Ice Center (NIC) which states, `U.S. National Ice Center (USNIC) is the global entity that names, tracks, and documents Antarctic icebergs that meet the criteria of 20 sqNM or greater, or 10 NM on its longest axis.`

So... missing all the small bergs.

If we have caling (from velocity and shelf front position) and tracked new icebergs (from the BYU obs), the difference are all the small bergs. These could, perhaps, be presumed to melt at the surface and immediately adjacent to the shelf (i.e., at their source).

There is another other observational datasets: https://scar.org/library-data/data/iceberg-database

**Reply by nicojourdain** [reply]
yes, and in Antarctica, small icebergs do not represent a large fraction of the total iceberg mass according to Tournadre et al. (2016):

<img width="405" alt="Screenshot 2024-05-02 at 22 50 20" src="https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/25052687/6b49da5e-216d-498d-a41a-c117ff3b56cc">

I am curious to see if the estimated thickness and area evolution is good enough to give a total melt rate that is consistent with the total calving rate.

**Reply by mankoff** [reply]
> yes, and in Antarctica, small icebergs do not represent a large fraction of the total iceberg mass according to Tournadre et al. (2016):

But we're not (only) interested in iceberg mass. We want iceberg meltwater flux.

The graph above (see https://github.com/NASA-GISS/freshwater-forcing-workshop/discussions/19#discussioncomment-9277472 on GitHub) displays different properties than melt which probably scales roughly linearly with surface area (not quite due to different depths).

In the *worst case* scenario where surface area is minimized as icebergs break apart and get smaller, an iceberg 10 % of the original size (1 order of magnitude decrease per the columns in the figure) leads to 22 % of the surface area. This means 22 % of the original melt even though the new iceberg is 10 % of the original size.

It's possible to have iceberg B with an order of magnitude less mass than iceberg A, but still have surface area of B > 0.22 * surface area of A, meaning higher melt rates.

Worst case scenario 2 order of magnitude mass decrease = 5 % of original melt. It could be much larger if surface area is larger.

The graph above also doesn't provide # of icebergs, just %, so I'm still not sure how to map the above numbers to that graphic, or other data from Tournadre (2016) http://doi.org/10.1002/2015jc011178

**Reply by mankoff** [reply]
Following up... I think that a 4 order of magnitude reduction in mass leads to (worst case) 0.2 % of the surface area (melt). From the above figure, looking at red (# of bergs), left column is ~100 %. Right column is ~1 %. That means melt from the left column (smallest) is approximately equal to melt from right column (largest). Am I doing something wrong here? Happy to share my work (code).

**Reply by mankoff** [reply]
My definition of 'worst case' is incorrect. I assumed an initial square berg (minimum surface area to volume ratio for a six-sided surface (e.g., not a sphere)), and then 'worst case' meant the smaller berg was also square. In reality, the initial berg is not likely to be square -> increased melt per unit volume. Smaller bergs could therefore have a lower area : volume ratio (more square) which would decrease their (relative) contribution, or a higher area : volume ratio which would increase their relative contribution.

But keeping things square (or spherical), it seems that all the small bergs may contribute roughly equal meltwater to the large bergs. I think...

I think this means we need to handle this - not ignore it. How it is handled is a function of how the iceberg product is derived.

For example,

+ If icebergs are a residual after say GRACE-derived grounded mass loss and observed sub-shelf melt-from-thinning, then this product contains all iceberg mass,  but NOT all bergs are big bergs. Some % of those bergs are small bergs near the ice front that likely melt immediately, at the shelf front, and at the surface. Some are created from the big bergs as they travel.

+ If the icebergs are from the observed BYU product that doesn't see small bergs, then we should double(?!) the estimate of icebergs, assuming small bergs contribute as much melt as big bergs. Many are created at shelf fronts from slump events, but many occur from the big bergs as they travel.

+ If the icebergs are from a model... not sure what to assume in this case. We'd need to know more about the model.

**Reply by mankoff** [reply]
Ah. Sorry for working this out in public, but I realized another limitation of that figure. It doesn't capture residence time (which I was getting at above with mention of melt rate).

The 1 % (by number, 70 % by area) big iceberg(s) may last a decade. The 75 % (by number, 1 % by area) small icebergs may last < 1 year, so > 10x as many per decade, in which case, again, the small ones could matter.

Anyway, I think more work is needed here, or someone else can offer an interpretation of this figure.

**Reply by nicojourdain** [reply]
sorry my label was wrong, black was iceberg area, not mass, I've corrected the label in my previous post. 

You are right, melt does not scale with the iceberg volume, it partly scales with the iceberg perimeter times thickness (lateral melt due to convective plumes and waves) and partly with the iceberg area (basal melt). The total iceberg area mostly consists of large icebergs. In terms of perimeter, an equal length would be represented by each bin of the histogram. This can be obtained by integrating area or perimeter weighted by Tournadre's power low. I view these distributions as a snapshot of the iceberg population, I don't think it tells anything about the fate of individual icebergs.

**Comment by gavinny** [comment]
Yes – I think we should just use this, but do we need to think about the conversion to total melt so that we can derive a weighting function? I don’t think that we need to worry too much about the small missing flux compared to the total.

Gavin

--
Gavin A. Schmidt
Director, NASA Goddard Institute for Space Studies
2880 Broadway, New York NY 10025
Cell: 212 749 0006
Email: ***@***.******@***.***>
URL: https://www.giss.nasa.gov/staff/gschmidt


From: Nicolas C. Jourdain ***@***.***>
Date: Tuesday, April 30, 2024 at 12:22 PM
To: NASA-GISS/freshwater-forcing-workshop ***@***.***>
Cc: Schmidt, Gavin A. (GISS-6110) ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] [BULK] Re: [NASA-GISS/freshwater-forcing-workshop] Icebergs (Discussion #19)
CAUTION: This email originated from outside of NASA.  Please take care when clicking links or opening attachments.  Use the "Report Message" button to report suspicious messages to the NASA SOC.



Anna Olivé-Abello, working with Pierre Mathiot and myself at IGE, has actually already estimated the melt patterns from the 18 IMBIE basins. We can probably average over a longer period to have a smoother pattern.
MAP_NEW1v1_BedMachine_meltingbasin_lr.png (view on web)<https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/25052687/810833f0-0a03-4844-8895-d25cd05bffbd>

This is not perfect as this is a circum-Antarctic domain cut at 55°S, so we are missing ~70Gt/yr of melt in the South Atlantic, but this is probably the best we have in our group without having to wait for new simulations (and we can combine previous patterns to fix the South Atlantic leakage if needed).

**Comment by twilamoon-science** [comment]
Noting this new publication for group awareness: https://gmd.copernicus.org/articles/17/3279/2024/

**Reply by trackow** [reply]
Adding @ackerlar here in case he has further input

**Comment by mankoff** [comment]
Thanks to introductions from the SOFIA group I've found @julianamarson, their paper 2024 http://doi.org/10.1029/2023jc020697 and the associated data at https://canwin-datahub.ad.umanitoba.ca/data/dataset/nemo-anha4-seaice-locking-icebergs/resource/8aa9c193-214e-4152-9abe-037010bf1999

This is still a work in progress but I think we now have a Greenland product complementing the graphic from @nicojourdain above. That is, meltwater from icebergs distributed around Greenland and tied back to a source location.

Work is ongoing at the `greenland_iceberg_melt.org` workbook in this repository: https://github.com/NASA-GISS/freshwater-forcing-workshop/blob/main/greenland_iceberg_melt.org
